devtools/platform-descriptor-json/src/main/resources/templates/basic-rest/scala/build.gradle-template.ftl

```diff
@@ -20,3 +20,6 @@ dependencies {
 group '${project_groupId}'
 version '${project_version}'
 
+compileScala {
+  scalaCompileOptions.encoding = 'UTF-8'
+}
```

Comment:
I see you added something for Scala. We don't need something for Kotlin?

Reply:
According to https://stackoverflow.com/questions/45564450/how-to-set-kotlin-source-encoding-in-gradle: 

>Kotlin source files are always UTF-8 by design. There is no way to specify any other encoding.